comp/netflow/goflowlib/netflowstate/netflow.go

```diff
@@ -381,3 +149,31 @@ func (s *StateNetFlow) FlowRoutine(workers int, addr string, port int, reuseport
 	s.initConfig()
 	return utils.UDPStoppableRoutine(s.stopCh, "NetFlow", s.DecodeFlow, workers, addr, port, reuseport, s.Logger)
 }
+
+func (s *StateNetFlow) sendTelemetryMetrics(msg any, exporterIP string) {
+	switch msgDec := msg.(type) {
+	case netflow.NFv9Packet:
+		s.sender.Count(common.MetricPrefix+"processor.processed", 1, "", []string{"exporter_ip:" + exporterIP, "version:9", "flow_protocol:netflow"})
+		for _, fs := range msgDec.FlowSets {
+			s.sendFlowSetMetrics(fs, exporterIP, "9")
+		}
+	case netflow.IPFIXPacket:
+		s.sender.Count(common.MetricPrefix+"processor.processed", 1, "", []string{"exporter_ip:" + exporterIP, "version:10", "flow_protocol:netflow"})
+		for _, fs := range msgDec.FlowSets {
+			s.sendFlowSetMetrics(fs, exporterIP, "10")
+		}
+	}
+}
+
+func (s *StateNetFlow) sendFlowSetMetrics(fs any, exporterIP string, version string) {
+	switch fs.(type) {
+	case netflow.TemplateFlowSet:
+		s.sender.Count(common.MetricPrefix+"processor.flowsets", 1, "", []string{"exporter_ip:" + exporterIP, "version:" + version, "type:template_flow_set", "flow_protocol:netflow"})
```

Comment:
Shall we try to avoid repeating some of the tags like `"exporter_ip:" + exporterIP, "version:" + version`, it's repeated a lot of time in `sendTelemetryMetrics` and `sendFlowSetMetrics`.

For example, you can create the tags `"exporter_ip:" + exporterIP, "version:9"` and `"exporter_ip:" + exporterIP, "version:10"` in `sendTelemetryMetrics` then pass the tags down to `sendFlowSetMetrics`

Reply:
Following discussions, I updated the logic to send metrics through Prometheus, to avoid having two different ways of sending telemetry